server/schema/dashboard.go

```diff
@@ -75,16 +78,22 @@ const (
 	DashboardElementTypeReportTemplate DashboardElementType = "report"
 	DashboardElementTypeGARealtime     DashboardElementType = "ga-realtime"
 	DashboardElementTypeNRServers      DashboardElementType = "nr-servers"
+	DashboardElementTypeTCBuilds       DashboardElementType = "tc-builds"
 )
 
 func (t DashboardElementType) Clean() DashboardElementType {
-	switch t {
-	case DashboardElementTypeReportTemplate, DashboardElementTypeGARealtime,
-		DashboardElementTypeNRServers:
+	validValues := map[DashboardElementType]bool{
+		DashboardElementTypeReportTemplate: true,
+		DashboardElementTypeGARealtime:     true,
+		DashboardElementTypeNRServers:      true,
```

Comment:
just to follow our approach you should revert this changes.

i am practically sure that `switch` is more faster than `map`, but you can write a benchmark if not.